### PR TITLE
Added docs for how to produce a custom Linux OS image

### PR DESCRIPTION
**Pull request by soccerGB**
This is the first documentation cut of the custom OS image build instructions for the Linux Hyper-V container on Windows.

### CODE REVIEW

File: docs/customosbuildinstructions.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+
+
+# How to produce a custom Linux OS image
+
+A LCOW custom Linx OS image was divided into two parts: a Linux kernel module and a set of user-mode componments. Both parts were highly customized for the purpose of supporting Linux Hyper-V container on Windows
```
Comment (gupta-ak) [inline docs/customosbuildinstructions.md]: Linx -> Linux

Comment (gupta-ak) [inline docs/customosbuildinstructions.md]: componments -> components

File: kernelconfig/4.11/scripts/init_script [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+export PATH = / usr / local / sbin: / usr / local / bin : / usr / sbin : / usr / bin : / sbin : / bin
```
Comment (gupta-ak) [inline kernelconfig/4.11/scripts/init_script]: still has weird spaces

File: docs/customosbuildinstructions.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+- Build your kernel 
+
+
+    Note:  The key delta between the upsteam default setting and above kconfig is in the area of ACPI/NIFT/NVDIMM/OverlyFS/9pFS/Vsock/HyerpV settings, which were set to be built-in instead of modules.
+           The Kconfig above is still a work in process in terms of trimming any unnecessary components from the kernel image.  
+
+## How to construct user-mode components
+
+THe expected user mode directory structure is required to constructed as follows: [See the complete user-mode file list](../kernelconfig/4.11/completeUsermodeFileLists.md/)
```
Comment (gupta-ak) [inline docs/customosbuildinstructions.md]: THe -> The

File: docs/customosbuildinstructions.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+             /bin/ip
+             /bin/iproute
+             /bin/hostname
+
+- Debugging tools: mostly from busybox tool set
+       
+
+# Supported LCOW custom Linux OS packaging formats
+
+    A LCOW custom Linux OS could be packaged into two different supported formats:
```
Comment (gupta-ak) [inline docs/customosbuildinstructions.md]: The markdown makes everything in here look in a code box. Is that intended?

File: docs/customosbuildinstructions.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+2. **/init** 
+   This is the [init script file](../kernelconfig/4.11/scripts/init_script)
+
+3. **/root** : this is the home directory of the root account. At this moment, it contains a sandbox file with a prebuilt empty ext4 fs for supporting Service VM operations
+         /root/integration/prebuildSandbox.vhdx
+
+4. **/sbin** : 
+        /sbin/runc  
+
+              Note:this is the "runc" binary for hosting the container execution environment.
```
Comment (gupta-ak) [inline docs/customosbuildinstructions.md]: Do you want the first two lines in the code box?

File: docs/customosbuildinstructions.md (hunk 3 above)
Comment (soccerGB) [inline docs/customosbuildinstructions.md]: good catch

**Pull request merged**